utils/tim_delay.c

```diff
@@ -0,0 +1,65 @@
+/*
+ * tim_delay.c
+ *
+ *  Created on: 19-Feb-2021
+ *      Author: Sonal Shrivastava
+ */
+
+#include <stdint.h>
+
+#include "drivers/mss_timer/mss_timer.h"
+#include "CMSIS/system_m2sxxx.h"
+#include "tim_delay.h"
+#include "led.h"
+
+
+// Delay counter
+//static volatile uint32_t DelayCounter;
+volatile uint32_t DelayCounter;
+
+
+void SysTick_Handler(void)
+{
+    DelayCounter++;
+}
+
+
+void tim_init(void)
```

Comment:
How about `delay_init_timer()`?

Reply:
I renamed the file from `tim_delay.c` to `timer_delay.c`. Hence, modified the functions' names to match the file name. `timer_delay_init()` and `timer_delay()`